RELEASE.md

```diff
@@ -0,0 +1,19 @@
+# How to release
+
+This project is hosted on NPM.  You can find the terra packages [here][project-url].
+
+Below is a guide for releasing packages:
+
+1. Update the versions of the component and change log via `npm run version:<major, minor, patch>`
+2. Create a Release PR, Get approvals
+3. Merge PR to `main`
+4. Watch main to ensure releases get pushed to npm and tagged in github appropriately
+
+If you are publishing a new package, e.g. a v1.0.0 release, the package will be listed with only you as the collaborator.
```

Comment:
Should we centrally locale this Release doc in terra-ui?

If not, this could be updated / removed since it's not a mono-repo

Reply:
lol, you're correct